x-pack/plugins/secops/public/components/timeline/body/sort/sort_indicator.tsx

```diff
@@ -4,11 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { EuiIcon } from '@elastic/eui';
 import * as React from 'react';
 import { pure } from 'recompose';
+import styled from 'styled-components';
+
 import { SortDirection } from '.';
 
-export type SortDirectionIndicator = '' | '^' | 'v';
+export type SortDirectionIndicator = undefined | 'sortUp' | 'sortDown';
```

Comment:
Note: I had to change this from empty string to `undefined` as that was what Eui was expecting.